__tests__/itinerary.test.js

```diff
@@ -0,0 +1,18 @@
+const Itinerary = require('../src/itinerary.js')
+
+describe('Itinerary class and properties', () => {
+   const port1 = jest.fn(), port2 = jest.fn();
+   let itinerary = new Itinerary([port1, port2]);
+   
+   describe('Itinerary class', () => {
+      it('can be instatiated', () => {
+      expect(itinerary).toBeInstanceOf(Object);
```

Comment:
indentation seems to be off here

you can use format document option within VS Code to format these 

<img width="464" alt="Screen Shot 2020-11-21 at 13 33 24" src="https://user-images.githubusercontent.com/1476886/99881017-08480000-2c0f-11eb-9a6c-68ddaef72633.png">